backend/substrapp/tasks/k8s_backend.py

```diff
@@ -384,7 +384,8 @@ def _k8s_build_image(path, tag, rm, cache_index):
 def k8s_get_image(image_name):
     response = requests.get(
         f'{REGISTRY_SCHEME}://{REGISTRY}/v2/{image_name}/manifests/substra',
-        headers={'Accept': 'application/json'}
+        headers={'Accept': 'application/json'},
+        timeout=30
```

Comment:
this value should be set as settings value so we can adapt it easily if it doesn't fit the need

Reply:
done: https://github.com/SubstraFoundation/substra-backend/pull/324/commits/0dc01bd267b265a0202811355b18e923994a1d67